Assignment/Assignment/SampleData.cs

```diff
@@ -1,30 +1,79 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 
 namespace Assignment
 {
     public class SampleData : ISampleData
     {
+        const int FirstName = 1;
+        const int LastName = 2;
+        const int EmailAddress = 3;
+        const int StreetAddress = 4;
+        const int City = 5;
+        const int State = 6;
+        const int Zip = 7;
+
         // 1.
-        public IEnumerable<string> CsvRows => throw new NotImplementedException();
+        public IEnumerable<string> CsvRows
+        {
+            get
+            {
+                List<string> result = new List<string>();
+                using (var fileReader = new StreamReader(@$".\People.csv"))
+                {
+                    for (string? s = fileReader.ReadLine(); !fileReader.EndOfStream; s = fileReader.ReadLine())
+                    {
+                        result.Add(s!);
+                    }
+                }
+                return result.Skip<string>(1);
+            }
+        }
 
         // 2.
         public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows() 
-            => throw new NotImplementedException();
+        {
+            SampleData sampleData = new SampleData();
+            IEnumerable<IPerson> personCollection = sampleData.People;

```

Comment:
```suggestion
            IEnumerable<IPerson> personCollection = People;
```
You don't need to create a new instance of the class if you're in that class.